VIMVideoPlayer/VIMVideoPlayer.m

```diff
@@ -734,6 +734,16 @@ - (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(N
                 }
             }
         }
+        else if (newStatus == AVPlayerItemStatusReadyToPlay)
```

Comment:
@huebnerob Consider adding a comment here describing why / what happens when we receive `newStatus == oldStatus` and it's `AVPlayerItemStatusReadyToPlay`. What does this mean?